nemo/collections/nlp/models/language_modeling/megatron_finetune_model.py

```diff
@@ -168,38 +169,10 @@ def on_test_epoch_start(self):
         )
         return super().on_test_epoch_start()
 
-    def on_test_epoch_end(self):
```

Comment:
@aklife97 this is not removed. We had two functions `test_epoch_end` and `on_test_epoch_end`. With lightning 2.0 since `test_epoch_end` callback is deprecated and renamed to `on_test_epoch_end`, I have merged the functionality of previous `test_epoch_end` into `on_test_epoch_end`. You can find the configuration of batch size being handled here in `inference_epoch_end` [https://github.com/NVIDIA/NeMo/blob/athitten/ptl_2.0/nemo/collections/nlp/models/language_modeling/megatron_finetune_model.py#L549](url)

Reply:
sounds good, thanks for the clarification!